src/book/handlers.ts

```diff
@@ -0,0 +1,49 @@
+import { PrismaClient } from '@prisma/client';
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { BookPayload } from './index';
+import { getBookService } from './book-service';
+
+export const getBooksHandlers = (prisma: PrismaClient) => {
+  const { getAllBooks, createBook, getBookById, deleteBook, updateBook } =
+    getBookService(prisma);
+
+  return {
+    getAll: async (_: FastifyRequest, reply: FastifyReply) => {
+      const books = await getAllBooks();
+
+      reply.send(books);
+    },
+    getById: async (
+      request: FastifyRequest<{ Params: { id: string } }>,
+      reply: FastifyReply,
+    ) => {
+      const book = await getBookById(request.params.id);
+
+      reply.send(book);
```

Comment:
I'm not sure if this is a great idea, but consider inlining the service method calls like so
```suggestion
      reply.send(await getBookById(request.params.id));
```

Reply:
I'm not sure either but I'll give it a try ;)